lib/version_utils.pm

```diff
@@ -290,11 +289,6 @@ sub is_x86_64 {
     return check_var('ARCH', 'x86_64');
 }
 
-sub is_remote_backend {
-    # s390x uses only remote repos
-    return is_s390x() || check_var('BACKEND', 'svirt') || check_var('BACKEND', 'ipmi') || check_var('BACKEND', 'spvm');
-}
-
```

Comment:
please remove one more blank line, use only one line for separating between methods :)